projects/plugins/boost/app/modules/critical-css/rest-api/BoostEndpoint.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Boost REST API Endpoint.
+ *
+ * @package automattic/jetpack-boost
+ */
+
+namespace Automattic\Jetpack_Boost\Modules\Critical_CSS\REST_API;
+
+interface BoostEndpoint {
```

Comment:
I assume this will get moved out of the `Critical_CSS` module at some point?

Reply:
Yeah, it should.